doc-app/tests/integration/components/ember-input-validation/tpk-validation-datepicker-test.ts

```diff
@@ -25,33 +25,7 @@ module('Integration | Component | tpk-validation-datepicker', function (hooks) {
     return changeset;
   }
 
-  test('It changes data-has-error attribue on error', async function (assert) {
-    const changeset = await setupComponent.call(this);
-    assert.dom('[data-test-tpk-datepicker]').exists();
-    assert.dom('[data-test-tpk-label]').containsText('label');
-
-    setTempusDominusDate('[data-test-tpk-datepicker-content]', '');
-    await settled();
-
-    assert.dom('[data-test-tpk-datepicker-content]').hasNoText();
-
-    assert
-      .dom('[data-test-tpk-datepicker]')
-      .hasAttribute('data-has-error', 'false');
-
-    changeset.addError({
-      message: 'required',
-      value: '',
-      originalValue: 'a',
-      key: 'date',
-    });
-
-    await settled();
-
-    assert.strictEqual(changeset.get('date'), null);
-
-    assert
-      .dom('[data-test-tpk-datepicker]')
-      .hasAttribute('data-has-error', 'true');
+  test('Dummy test', async function (assert) {
+    assert.expect(0);
```

Comment:
Il est vraiment pertinent lui ? :p

Reply:
je sais, je l'ai laissé car on va sûrement réécrire des tests dedans très bientôt

Reply:
ok fine for me